assets/src/block-editor/components/amp-preview.js

```diff
@@ -129,6 +139,23 @@ class AMPPreview extends Component {
 		if ( previewLink && ! prevProps.previewLink ) {
 			this.setPreviewWindowLink( previewLink );
 		}
+
+		this.moveButton();
+	}
+
+	/**
+	 * Moves the (non-AMP) 'Preview' button to before this 'Preview AMP' button, if it's not there already.
+	 */
+	moveButton() {
```

Comment:
Calling this in the 'Preview AMP' button's `componentDidUpdate()` method moves it back to the correct place.

Without this, here's what it looks like:

![amp-render-moved](https://user-images.githubusercontent.com/4063887/65919169-20bce180-e3a1-11e9-9ced-ae9aed800894.gif)